playbooks/hc-ansible-deployment/he_gluster_vars.json

```diff
@@ -1,4 +1,5 @@
 {
+  // The passwords variables should be encryped using ansible-vault
   "he_appliance_password": "root-password-of-host",
```

Comment:
perhaps we could include the encryption warning in the dummy password.

Reply:
@rohantmp no. We to somehow convey the user that it has to be encrypted. Is there any way at all to add comments in json? Like "_comment": ... something like that?